### DISCUSSION TITLE
Split Tall image into segments to remove whitespace and then reappend

### DISCUSSION THREAD

**Discussion by bcpletcher** [ImageMagick/ImageMagick#4236] [Help]
I have a comic strip that is 8000px tall by 720px wide with large amounts of whitespace between each panel. What would be the best way to remove the whitespace between them (possibly leave a 30px gap?). Also some panels are fully on the right, center, or left with white space on the sides. Can I trim just the top and bottom?

My thought process was to split the file into 1px high segments, strip whitespace, and them append them back together. Is this possible at all? Im working through a variety of commands but nothing seems to work.

So far I have `convert test.png +repage -crop 1x500@ -gravity South -background white -splice 0x1 -background black -splice 0x1 -trim -chop 0x1 -background white -append out.png`

Currently im only dividing it by 500 equal pieces but would like to get more fine tuned but am not sure how, or if I can optimize this script in any way

Any help would be appreciated!

**Comment by snibgo** [comment]
What version of IM, on what platform?

Please show a sample input.
